h/groups/views.py

```diff
@@ -124,8 +124,15 @@ def _read_group(request, group):
     document_links = [annotation.document_link
                       for annotation in annotations_by_uri.values()]
 
+    member_usernames = [member.username
+                        for member in group.members]
+
     template_data = {
-        'group': group, 'group_url': url, 'document_links': document_links}
+        'group': group,
+        'group_url': url,
+        'document_links': document_links,
+        'member_usernames': sorted(member_usernames)
```

Comment:
I'm pretty sure you don't even need this change, do you? Just pass in group and then in the templates iterate over the users as:

```jinja2
{% for user in group.members|sort(attribute='username') %}
...
{% endfor %}
```